src/version.h

```diff
@@ -9,7 +9,7 @@
  * network protocol versioning
  */
 
-static const int PROTOCOL_VERSION = 170050;
+static const int PROTOCOL_VERSION = 170100;
```

Comment:
I confirmed that this matches the values in `src/chainparams.cpp` and [ZIP 252](https://zips.z.cash/zip-0252).